docs/docs/index.md

```diff
@@ -0,0 +1,73 @@
+Pathfora JS is a lightweight SDK for displaying personalized modules on your website. it integrates with your [Lytics](http://www.getlytics.com/) account to track user actions, and conditionally display modules based on your users' audience memebership. This documentation will walk you through everything you need to create your own highly customized and personalized module. Code and live preview examples are provided along the way.
+
+## Getting Started
+Before you begin creating modules with Pathfora you must have add the [Lytics Javascript Tag](https://activate.getlytics.com/#/documentation/jstag_anon) to your website. The Pathfora tag must be loaded after the Lytics tag.
+
+``` html
+<!-- Your Lytics JS Tag -->
+<script src="https://api.lytics.io/api/tag/{{YOUR LYTICS API KEY}}/lio.js"></script>
+
+<!-- Pathfora Tag -->
+<script src="http://c.lytics.io/static/pathfora.min.js"></script>
+```
+
+Next you'll need to set up configuration for your module and initialize it with `pathfora.initializeWidgets` example configurations, what all these settings mean, and their outputted modules can be found throughout the documentation.
+
+## Overview
+Here are a couple sections to get you started on the basics of Pathfora JS. See the navigation in the sidebar for a full list of docs.
+
+- **[Types](types/message.md)** 
+> The type parameter relays information to the layout (see below) related to the type of content being rendered. Pathfora currently supports four types of modules: message, form, subscription and gate.
```

Comment:
This is super obnoxious of me, but I have to say it. You are creating blockquotes here for the purpose of indentation. Since these aren't quotes, they should be marked down in some other way.